### PR TITLE
feat: [FW-12] Create main layout

### PR DESCRIPTION
**Pull request by barroro**
Adds the main components for the root layout
- Add the Header component
    - The header is not responsive and the navigation is hidden depending on the viewport. In the next iteration, I will add a dropdown for mobile screens. 
- Add the Main component to wrap the dynamic content loaded in each page
- Add the Footer component

Note: Navigation is not working yet, we need to define all the routes and create the component for each page

### CODE REVIEW

File: frontend/src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          <nav className="flex gap-10 font-medium text-sm">
+            <Link href="/get-started">Get started</Link>
+            <Link href="/how-it-works">How does it works?</Link>
+            <Link href="/about-us">About us</Link>
+            <Link href="/terms-of-service">Terms of Service</Link>
+            <Link href="/privacy-policy">Privacy Policy</Link>
+          </nav>
+        </div>
+        <div className="mt-8 text-center text-sm">
+          <p>&copy; 2021 Festwrap. All rights reserved.</p>
```
Comment (DanielMoraDC) [inline frontend/src/components/Footer.tsx:17]: Probably 2024, right? 😅

Comment (barroro) [reply frontend/src/components/Footer.tsx:17]: Yes, I've generated some of the code with v0 to try the platform and I didn't see this mistake. I change it now.

**Review by DanielMoraDC — APPROVED** [review]
Good job! I really appreciate the atomic commits, they make PRs super easy to review 🚀 

Only one thing: I see that, on 404 error, the background becomes dark and the logo is difficult to see. We might want to iterate on that in the near future.

Feel free to self-resolve the tiny comment I left.

**Comment by DanielMoraDC** [comment]
Manually merging for now, while we resolve the signing issue.

**Pull request merged**